### PR TITLE
add macos_support: try to fix the 'cannot find camera issue'

### PR DESCRIPTION
**Pull request by ruizhaogit**
Hi guys,

I send you a pull request and try to fix the issue [Unable to find sensor: camera, Segmentation fault: 11](https://github.com/HorizonRobotics/SocialRobot/issues/20).
@Jialn Please test it on MacOS and consider merging if it works.

Best,
Rui

### CODE REVIEW

**Comment by Jialn** [comment]
Thanks, it works pretty well. 
Pls remove "gazebo::sensors::SensorManager* mgr = gazebo::sensors::SensorManager::Instance();"
as we don't need *mgr any more.

File: pygazebo/pygazebo.cc [hunk 1]
```diff
@@ -201,7 +201,7 @@ class Agent : public Model {
 
       gazebo::sensors::ContactSensorPtr sensor =
           std::dynamic_pointer_cast<gazebo::sensors::ContactSensor>(
-              mgr->GetSensor(contact_sensor_name));
```
Comment (Jialn) [inline pygazebo/pygazebo.cc]: Pls remove "gazebo::sensors::SensorManager* mgr = gazebo::sensors::SensorManager::Instance()" before this line.

File: pygazebo/pygazebo.cc [hunk 2]
```diff
@@ -233,7 +233,7 @@ class Agent : public Model {
 
       gazebo::sensors::CameraSensorPtr sensor =
           std::dynamic_pointer_cast<gazebo::sensors::CameraSensor>(
-              mgr->GetSensor(sensor_scope_name));
```
Comment (Jialn) [inline pygazebo/pygazebo.cc]: Pls remove "gazebo::sensors::SensorManager* mgr = gazebo::sensors::SensorManager::Instance()" before this line.

File: pygazebo/pygazebo.cc (hunk 1 above)
Comment (ruizhaogit) [reply pygazebo/pygazebo.cc]: It is removed :)

File: pygazebo/pygazebo.cc (hunk 2 above)
Comment (ruizhaogit) [reply pygazebo/pygazebo.cc]: It is removed :)

**Pull request merged**